zb-db/src/test/java/io/camunda/zeebe/db/impl/DbTransactionTest.java

```diff
@@ -476,10 +477,15 @@ public void shouldNotWrapRuntimeExceptions() {
         .isSameAs(exception);
   }
 
-  private enum ColumnFamilies {
+  private enum ColumnFamilies implements EnumValue {
     DEFAULT, // rocksDB needs a default column family
     ONE,
     TWO,
-    THREE
+    THREE;
+
+    @Override
+    public int getValue() {
+      return ordinal();
```

Comment:
:thinking_face: Could also be the default

Reply:
I dislike that because it makes it easier to overlook. Here it's fine because this is an enum only used in tests.